testingdocs.md

```diff
@@ -0,0 +1,187 @@
+
+
+
+
+## location type(bims.models.location_type.LocationType)
+
+```
+Location type model.
+```
+
+|Fullname|Name|Type|PK|Unique|Index|Null/Blank|Comment|
+|---|---|---|---|---|---|---|---|
+|ID |id |serial |True |True | |Blank | |
+|name |name |varchar(100) | | | | | |
+|description |description |text | | | |Both | |
+|allowed geometry |allowed_geometry |varchar(20) | | | | |POINT:Point, LINE:Line, POLYGON:Polygon, MULTIPOLYGON:Multipolygon |
+
+Options
+```
+ordering : ['name']
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## location context(bims.models.location_context.LocationContext)
+
+```
+LocationContext(id, context_document)
+```
+
+|Fullname|Name|Type|PK|Unique|Index|Null/Blank|Comment|
+|---|---|---|---|---|---|---|---|
+|ID |id |serial |True |True | |Blank | |
+|context document |context_document |text | | | | | |
+
+Options
+```
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## location site(bims.models.location_site.LocationSite)
+
+```
+Location Site model.
+```
+
+|Fullname|Name|Type|PK|Unique|Index|Null/Blank|Comment|
+|---|---|---|---|---|---|---|---|
+|ID |id |serial |True |True | |Blank | |
+|name |name |varchar(100) | | | | | |
+|location type |location_type |integer | | |True | |FK:bims.models.location_type.LocationType |
+|geometry point |geometry_point |geometry(POINT,4326) | | | |Both | |
+|geometry line |geometry_line |geometry(LINESTRING,4326) | | | |Both | |
+|geometry polygon |geometry_polygon |geometry(POLYGON,4326) | | | |Both | |
+|geometry multipolygon |geometry_multipolygon |geometry(MULTIPOLYGON,4326) | | | |Both | |
+|location context |location_context |integer | | |True |Both |FK:bims.models.location_context.LocationContext |
+
+Options
+```
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## IUCN Status(bims.models.iucn_status.IUCNStatus)
+
+```
+IUCN status model.
+```
+
+|Fullname|Name|Type|PK|Unique|Index|Null/Blank|Comment|
+|---|---|---|---|---|---|---|---|
+|ID |id |serial |True |True | |Blank | |
+|category |category |varchar(50) | | | |Blank |LC:Least Concern, NT:Near Threatened, VU:Vulnerable, EN:Endangered, CR:Critically Endangered, EW:Extinct In The Wild, EX:Extinct |
+|sensitive |sensitive |boolean | | | |Blank | |
+
+Options
+```
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## Taxon(bims.models.taxon.Taxon)
+
+```
+Taxon model.
+```
+
+|Fullname|Name|Type|PK|Unique|Index|Null/Blank|Comment|
+|---|---|---|---|---|---|---|---|
+|ID |id |serial |True |True | |Blank | |
+|GBIF id |gbif_id |integer | | | |Both | |
+|iucn status |iucn_status |integer | | |True |Both |FK:bims.models.iucn_status.IUCNStatus |
+|common name |common_name |varchar(100) | | | |Blank | |
+|scientific name |scientific_name |varchar(100) | | | |Blank | |
+|author |author |varchar(100) | | | |Blank | |
+
+Options
+```
+default_permissions : ('add', 'change', 'delete')
+```
+
+
+## survey-locationsite relationship(bims.models.survey.Survey_sites)
```

Comment:
I think you need to put proper verbose_name under meta class in all of these models, so it can generate this doc better.

Reply:
Alright @dimasciput let me revisite the bims models

Reply:
Hi @dimasciput 
Its okay now, with .rst we have all the models correctly named.
```## survey-locationsite relationship(bims.models.survey.Survey_sites)```
Just defines the relationship on the two models.
Thanks